resources/js/src/app/components/itemList/filter/ItemFilter.js

```diff
@@ -31,6 +37,7 @@ Vue.component("findologic-item-filter", {
 
     created() {
         this.$options.template = this.template || "#vue-item-filter";
+        this.facetType = (typeof this.facet.findologicFilterType !== 'undefined') ? this.facet.findologicFilterType : this.facet.type;
```

Comment:
When is this `undefined`? Is this the case when Findologic is not active?

Reply:
If I remember correctly, this happens when the shopkey is correct but Findologic navigation filters are disabled.

Reply:
When the Findologic navigation filters are disabled, the Plentymarkets filter should be shown, is this the case and can these be properly selected? If this is the case, this comment can be resolved, otherwise please investigate.